azure-quantum/azure/quantum/target/microsoft/target.py

```diff
@@ -393,7 +393,12 @@ def submit(self,
                 (module, _) = to_qir_module(input_data, record_output=False)
                 input_data = module.bitcode
         finally:
-            return super().submit(input_data, name, input_params, **kwargs)
+            return super().submit(
```

Comment:
since we're are not explicitly specify `shots` in the `submit()` function would Python make sure to pass the `shots` down to the generic `Target` as part of `**kwargs`?
Even though the generic `Target` doesn't use `shots` yet, I think it would make sense the `shots` still get passed correctly.
Could you please add unit-test for this case?

Reply:
Ah, yes, it definitely must be passed to the base submit method, in case we want to additionally handle this value in future in this base class. 
Since it is also derived from this base target, I think it just has to be explicit as anywhere else and not in `kwargs`, what do you think?

> Could you please add unit-test for this case?

I am not sure if we should add such test for a specific Target (for example this one), since I can't see much value from that, especially when it is explicitly defined.  Ideally, it would be nice to have such test, and not for a particular target but rather as a special test that exercises the "uniformity" of all existing targets. Will try to implement such test, in next commit, if possible.